azure-pipelines.yml

```diff
@@ -0,0 +1,115 @@
+# Mutilated beyond recognition from the example at:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/go
+
+trigger:
+- v2
```

Comment:
When this merges into master, will we just remove the trigger section?

Reply:
Yep. Or change it to `master` [to rebuild on direct commits](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml), which answers my reply in [here](https://github.com/caddyserver/caddy/pull/2768#discussion_r329156849).